raven/raven.go

```diff
@@ -70,6 +70,7 @@ type Event struct {
 	ServerName string                 `json:"server_name"`
 	StackTrace stacktrace.StackTrace  `json:"stacktrace"`
 	Http       *Http                  `json:"request"`
+	TargetDsn  string                 `json:"targetDsn"`
```

Comment:
Oh? If you can override the target DSN like this, then you don't need multiple clients at all